spec/services/post_alerter_spec.rb

```diff
@@ -343,6 +344,47 @@ def create_post_with_alerts(args = {})
     end
   end
 
+  context '@here' do
```

Comment:
There are a couple of test cases that I think will be good for us to add: 

1. Test that `SiteSetting.max_here_mentioned` is respected. 
1. Test for the `can_mention_here?` guardian check